### DISCUSSION TITLE
With multi-tenancy: how to change the logo url to point to the current tenant ?

### DISCUSSION THREAD

**Discussion by amoktar** [filamentphp/filament#11308] [Help]
### Package

Panel builder

### Package Version

v3.2.27

### How can we help you?

With multi-tenancy: how to change the logo url to point to the current tenant ?
Thanks

I'm using view as brandLogo
```php
return $panel
        // ...
        ->brandLogo(fn () => view('filament.admin.logo'));
```

with in the view file `Filament::getTenant()` works as expected and get the current tenant 
while `Filament::getUrl()` doesn't get the url for the currnet tenant 

Thanks

**Comment by timwassenburg** [comment]
I'm trying to solve the same issue. Did you find a solution, @AMoktar ?

As a workaround, I overwrote `vendor/filament/filament/resources/views/components/sidebar/index.blade.php` by moving it to `resources/views/vendor/filament-panels/components/sidebar/index.blade.php` and changed the URL of the logo to the current tenant dashboard.

**Comment by badrshs** [comment]
For anyone experiencing this issue later, you can simply use  
`->homeUrl("/")` in the panel provider.

Also, you don't really need to return a view to show the logo unless you need something very very specific.  

My current code is:

```php
->brandLogo("/img/header-logo.svg")
->brandLogoHeight('3rem')
->homeUrl("/")
```
But I agree, Documentation for this can be better
